src/app/technical-record/technical-record.component.html

```diff
@@ -383,12 +398,60 @@ <h1 class="govuk-heading-xl title">Technical record</h1>
             </mat-expansion-panel>
           </div>
 
+          <div class="auto-pad" [@.disabled]="true">
+            <mat-expansion-panel class="custom-header mat-elevation-z0 " hideToggle="true" #applicant
```

Comment:
As agreed, @irina005  had the changed implemented and waiting for ticket to refactor when possible.

reference: https://jira.dvsacloud.uk/browse/CVSB-15024